src/README.md

```diff
@@ -2,6 +2,34 @@
 
 This directory contains the source of ClusterFuzz.
 
+## 🐍 Project Setup (`asdf`)
```

Comment:
I'm not sure I got the reason for this. It's not overlapping with the Pipfile purposes?